study.md

```diff
@@ -32,24 +34,32 @@ array.
   # your answer here
 ```
 
+## Class#method!
+
 What does an exclamation point after a method usually signify?
 
 ```ruby
   # your answer here
 ```
+
+## Instantiation
 How do you create a class in Ruby?
 
 ```ruby
   # your answer here
 ```
 
+## Class Modification
+
 If I modify a class in Ruby will already created instances of that class have
```

Comment:
reflow 

`If I modify a class in Ruby, will previously created instances of that class have the methods I added?`

something like this maybe?